design model/components.model.json

```diff
@@ -194,18 +194,9 @@
                     "defaultValue": ""
                 },
                 {
-                    "name": "Months",
```

Comment:
This is used to define the number of months displayed in the calendar, in order to describe a multi-view datepicker. It should not be removed. If the naming is not clear enough it can be named "MonthViews" instead.